src/main/java/org/folio/circulation/domain/Request.java

```diff
@@ -95,7 +96,7 @@ public JsonObject asJson() {
   }
 
   boolean isFulfillable() {
-    return getFulfilmentPreference() == HOLD_SHELF;
+    return getFulfilmentPreference() == HOLD_SHELF || getFulfilmentPreference() == DELIVERY;
```

Comment:
As these are both of the valid chooses for fulfilment preference, does this check need to performed any more at all?

(A request with an invalid fulfilment preference should be rejected)

Reply:
There is NONE item in the enum. I don't see if it's rejected somewhere. So I would live the condition